ruby/Rakefile

```diff
@@ -129,10 +129,14 @@ task push_to_rubygems: [
   :package,
   :"release:guard_clean"
 ] do
-  puts "TODO: DISABLED PUSHING TO RUBYGEMS"
-  # abort unless system("gem push pkg/libdatadog-#{Libdatadog::VERSION}.gem")
-  # abort unless system("gem push pkg/libdatadog-#{Libdatadog::VERSION}-x86_64-linux.gem")
-  # abort unless system("gem push pkg/libdatadog-#{Libdatadog::VERSION}-aarch64-linux.gem")
+  [
+    "gem push pkg/libdatadog-#{Libdatadog::VERSION}.gem",
+    "gem push pkg/libdatadog-#{Libdatadog::VERSION}-x86_64-linux.gem",
+    "gem push pkg/libdatadog-#{Libdatadog::VERSION}-aarch64-linux.gem"
+  ].each do |command|
+    puts "Running: #{command}"
+    abort unless system(command)
```

Comment:
I hope that STDOUT of the Github flow will show what's wrong with command if it fails. But if not, maybe we can add some output here